libs/dh/message-archive/feature-search/src/lib/table.component.ts

```diff
@@ -0,0 +1,123 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, model, output } from '@angular/core';
+import { TranslocoDirective } from '@ngneat/transloco';
+
+import { VaterUtilityDirective } from '@energinet-datahub/watt/vater';
+import { WattButtonComponent } from '@energinet-datahub/watt/button';
+import { WattDataTableComponent } from '@energinet-datahub/watt/data';
+import { WattDatePipe } from '@energinet-datahub/watt/date';
+import { WattEmptyStateComponent } from '@energinet-datahub/watt/empty-state';
+import { WattTableColumnDef, WATT_TABLE } from '@energinet-datahub/watt/table';
+
+import {
+  GetArchivedMessagesQueryVariables,
+  SortEnumType,
+} from '@energinet-datahub/dh/shared/domain/graphql';
+import { GetArchivedMessagesDataSource } from '@energinet-datahub/dh/shared/domain/graphql/data-source';
+import { ArchivedMessage } from '@energinet-datahub/dh/message-archive/domain';
+
+@Component({
+  selector: 'dh-message-archive-search-table',
+  standalone: true,
+  imports: [
+    TranslocoDirective,
+    VaterUtilityDirective,
+    WATT_TABLE,
+    WattButtonComponent,
+    WattDataTableComponent,
+    WattDatePipe,
+    WattEmptyStateComponent,
+  ],
+  template: `
+    <watt-data-table
+      *transloco="let t; read: 'messageArchive'"
+      vater
+      inset="ml"
+      [searchLabel]="t('searchById')"
+      [error]="dataSource.error"
+      [ready]="dataSource.called"
+      (clear)="reset()"
+    >
+      <h3>{{ t('results') }}</h3>
+      <watt-button variant="secondary" icon="plus" (click)="new.emit()">
+        {{ t('new') }}
+      </watt-button>
+      <watt-table
+        *transloco="let resolveHeader; read: 'messageArchive.columns'"
+        #table
+        description="Search result"
+        [dataSource]="dataSource"
+        [columns]="columns"
+        [loading]="dataSource.loading"
+        [resolveHeader]="resolveHeader"
+        [activeRow]="selection()"
+        (rowClick)="selection.set($event)"
+      >
+        <ng-container *wattTableCell="columns['documentType']; let row">
+          <div>
+            {{ row.businessTransaction }}
+            <br />
+            <span class="number">{{ row.documentType }}</span>
+          </div>
+        </ng-container>
+        <ng-container *wattTableCell="columns['sender']; let row">
+          <div>
+            {{ row.sender?.displayName }}
+            <br />
+            <span class="number">{{ row.sender?.glnOrEicNumber }}</span>
```

Comment:
nitpick: the `number` styling is missing

Reply:
Well spotted! It probably needs some styling, but I'm awaiting feedback on this, since I'm not sure if the gln number has to be included here. Fixed in 4f4d8e8eefc3f553e7209458fdea72b79e07eea6